src/CoffeeShop/CoffeeShopDistance.php

```diff
@@ -0,0 +1,32 @@
+<?php
+namespace CoffeeShop;
+use base\ILocation;
+
+class CoffeeShopDistance extends CoffeeShop{
```

Comment:
Name for this was a little confusing, you probably meant `CoffeShopWithDistance`